invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/communities_items/ComputerTabletCommunitiesItem.js

```diff
@@ -0,0 +1,77 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2022 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import _get from "lodash/get";
+import React from "react";
+import { Image } from "react-invenio-forms";
+import { Button, Icon, Item, Label } from "semantic-ui-react";
+
+export const ComputerTabletCommunitiesItem = ({
+  result,
+  index,
+  community_type,
+  visibilityColor,
+  visibilityText,
+  visibilityIcon
```

Comment:
all of these props should be removed, we need only to accept result to simplify the interface of this component